### DISCUSSION TITLE
Are you migrating to goja?

### DISCUSSION THREAD

**Discussion by aadog** [livebud/bud#323] [Q&A]
Are you migrating to goja?

**Comment by matthewmueller** [comment]
Not entirely! I want to provide the option via: https://github.com/livebud/js

The calculation to consider is: V8 evaluation + CGO overhead vs. Goja evaluation. 

Unfortunately there's not a clear winner. It depends on the complexity of the script. This is why we'll need options.

**Reply by aadog** [reply]
What if we use dll calls cgo-less, I can wrap v8, windows support is the first

**Reply by aadog** [reply]
If you can compile, I think goja is the first choice. It supports windows across platforms
